website/preprints/model.py

```diff
@@ -178,16 +178,17 @@ def set_published(self, published, auth, save=False):
 
     def set_preprint_license(self, license_detail, auth, save=False):
 
-        set_license(self, license_detail, auth, node_type='preprint')
-
-        self.node.add_log(
-            action=NodeLog.PREPRINT_LICENSE_UPDATED,
-            params={
-                'preprint': self._id
-            },
-            auth=auth,
-            save=False
-        )
+        license_record, license_changed = set_license(self, license_detail, auth, node_type='preprint')
+
+        if license_changed:
+            self.node.add_log(
+                action=NodeLog.PREPRINT_LICENSE_UPDATED,
+                params={
+                    'preprint': self._id
+                },
```

Comment:
Discussed with @saradbowman/Product and agreed upon new log language for license updates, added [here](https://github.com/CenterForOpenScience/osf.io/pull/6633/commits/46d4326636eaa31840780603b070b12c95caa1c7)